static/bootstrap-responsive.css

```diff
@@ -23,8 +23,20 @@
         padding: 0;
     }
 
-    .span12{
-        max-width: 100%;
+    .span12, .span9, .footnotes {
```

Comment:
I'm a bit concerned about overriding the span sizes given that they are already defined by bootstrap. It seems like it would be better to make sure that the content are within the right CSS class rather than modifying the styles here.

Reply:
I understand your concern. Basically, both span12 and span9 classes could be easier to read on tablets using the maximum device screen size. Unfortunately, bootstrap 2.x only define fixed sizes for span classes whereas in bootstrap 3+ specific [grid classes](https://getbootstrap.com/docs/4.0/layout/grid/) can be associated to the same div to specify different span sizes according to device type. I tried to overcame this limitation, but we can apply a different/better solution.

Reply:
Sure. I just ported over the materials from the website circa 2014. If upgrading Bootstrap would fix the problem we may just want to do that.

Reply:
I updated the page title and the CSS rule to separate span12 and span9. Then, if you agree, we can work together on a different branch to update Bootstrap from v2.0.3 (current) to v4.6 (more stable) or v5.0 (latest).